lib/connection/connection_config.js

```diff
@@ -959,7 +976,7 @@ function createParameters()
     },
     {
       name: PARAM_RETRY_SF_STARTING_SLEEP_TIME,
-      defaultValue: 0.25,
+      defaultValue: 1,
```

Comment:
The login requests use a base time of 4, check out this [comment ](https://github.com/snowflakedb/snowflake-sdks-drivers-issues-teamwork/issues/575#issuecomment-1760362338) for more details

Reply:
Changed to 4.